tests/schema-validate.test.ts

```diff
@@ -49,7 +49,8 @@ test('Invalid JSON schema in creationTransition input condition must trigger an
     name: 'test',
     description: 'test',
     statuses: { new: {} },
-    creationTransition: { type: 'manual',
+    creationTransition: {
+      type: 'manual',
```

Comment:
If there is a newline after the `{`, there should be a bracket before the `}`.

The rule is to be consistent (https://eslint.org/docs/latest/rules/object-curly-newline), the error reporting is not great but I think the rule makes sense.